tests/tx/test_scripts.py

```diff
@@ -972,3 +982,105 @@ def test_get_sigops_count(self):
         self.assertEqual(get_sigops_count(input_script.data, p2pkh_script), 1)
         # if no output_script, count only input
         self.assertEqual(get_sigops_count(input_script.data), 1)
+
+    def test_op_sighash_bitmask(self) -> None:
+        with self.assertRaises(MissingStackItems):
+            op_sighash_bitmask(ScriptContext(stack=[], extras=Mock(), logs=[], settings=Mock()))
+
+        with self.assertRaises(MissingStackItems):
+            op_sighash_bitmask(ScriptContext(stack=[b''], extras=Mock(), logs=[], settings=Mock()))
+
+        with self.assertRaises(AssertionError):
```

Comment:
Why would it be an assertion error? Is it because they are not bytes?

Reply:
Exactly!